Compression.md

```diff
@@ -58,6 +58,10 @@ formats) defined by [RFC 1952](https://tools.ietf.org/html/rfc1952).
 If any ambiguity arises when implementing this format, the implementation
 provided by the [zlib compression library](https://zlib.net/) is authoritative.
 
+Readers should support reading pages containing multiple GZIP members, however,
```

Comment:
Might be worth linking to the RFC of GZIP members: https://datatracker.ietf.org/doc/html/rfc1952#section-2.2

Reply:
its already mentioned:
### GZIP
A codec based on the GZIP format (not the closely-related "zlib" or "deflate"
formats) defined by [RFC 1952]**([](https://tools.ietf.org/html/rfc1952)).**

Reply:
Right to clarify I'm suggesting that we link "members" to its specific section where it is defined.